health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/CustomAreaChart.js

```
@@ -0,0 +1,359 @@
+      })
+    });
+    return Array.from(mergeMap.values());
+  }
+  const chartData = useMemo(() => {
+    if (response?.responseData?.data?.length == 1) {
+      setmanageChart("Area");
+      if (id !== "fsmCapacityUtilization") {
+        return response?.responseData?.data?.[0]?.plots;
+      }
+      return response?.responseData?.data?.[0]?.plots.map((plot) => {
+        const [month, year] = plot?.name.split("-");
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)